### ISSUE TITLE
Missing file section 3.1

### ISSUE THREAD

**Issue by csdantas** [magnuspalmblad/averageRegions#6]
Cannot find the example file of section 3.1, the .csv file that should contain the name of ROIs

**Comment by magnuspalmblad** [comment]
Yes, this file is missing. Is there a preferred format for this file? If regions can have commas in their name, perhaps tab-separated file would be preferable? Or would we like to read in a file defining (new) regions by polygonal coordinates?

**Comment by csdantas** [comment]
I think the second option would be preferable, as I normally attribute a class to my ROIs and then export them from there.

**Comment by magnuspalmblad** [comment]
Perhaps the default behavior could be:
No input: export ALL existing ROIs as in the SCiLS Lab file
Input a list of names of ROIs: export ONLY those ROIs that are in the list
Input a list of names and polygons: generate new/overwrite existing ROIs and export those.

Do you have an example of a CSV file with exported regions?

**Comment by csdantas** [comment]
Agree.
Here is an example: 
[EpithelialROIs_test.csv](https://github.com/user-attachments/files/16052805/EpithelialROIs_test.csv)

**Comment by magnuspalmblad** [comment]
This ROI seems to be a list of pixel coordinates, whereas SCiLS Lab also defines ROIs as polygons, where the coordinates are the vertices. We need to think about if and how to support both of these. For now, averageRegions averages and outputs all regions already defined in the SCiLS Lab file, but ignores the first input.
